cms/templates/js/show-correctness-editor.underscore

```diff
@@ -0,0 +1,33 @@
+<form>
+<h3 class="modal-section-title" id="show_correctness_label"><%- gettext('Assessment Results Visibility') %></h3>
+<div class="modal-section-content show-correctness">
+    <div role="radiogroup" class="list-fields" aria-labelledby="show_correctness_label">
```

Comment:
OK, I played around with some example HTML myself. In order to get this to work (at least on the Mac with VoiceOver), you need to make this `role="group"` instead of using `radiogroup`.